src/popper/scm.py

```diff
@@ -167,10 +167,17 @@ def get_sha(repo, short=None):
     if not repo:
         return None
 
-    if short:
-        return repo.git.rev_parse(repo.head.object.hexsha, short=short)
+    try:
+        if short:
+            sha = repo.git.rev_parse(repo.head.object.hexsha, short=short)
+        else:
+            sha = repo.git.rev_parse(repo.head.object.hexsha)
+
+    except ValueError as e:
+        sha = None
+        log.warning(str(e))
```

Comment:
let's make this more friendly by showing a less git-cryptic message, something like `Could not obtain commit ID (SHA1) due to the Git repository at /path/to/repo being empty.`